hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ECContainerReplicaCount.java

```diff
@@ -68,16 +70,27 @@ public class ECContainerReplicaCount {
   private final Map<Integer, Integer> healthyIndexes = new HashMap<>();
   private final Map<Integer, Integer> decommissionIndexes = new HashMap<>();
   private final Map<Integer, Integer> maintenanceIndexes = new HashMap<>();
+  private final Set<ContainerReplica> replicas;
 
   public ECContainerReplicaCount(ContainerInfo containerInfo,
-      Set<ContainerReplica> replicas, List<Integer> indexesPendingAdd,
-      List<Integer> indexesPendingDelete, int remainingMaintenanceRedundancy) {
+      Set<ContainerReplica> replicas,
+      final ContainerReplicaPendingOps pendingOps,
```

Comment:
I don't think we should pass the instance of ContainerReplicaPending ops in here, as it makes this class dependent on it. Rather we should pass in the list of ops which were returned from the ContainerReplicaPendingOps instance in RM. Ie the parameter is List<ContainerReplicaOp>.

Reply:
I updated it to take List<ContainerReplicaOP>